src/System.Private.CoreLib/src/System/IO/FileLoadException.CoreCLR.cs

```diff
@@ -21,12 +20,12 @@ private FileLoadException(string fileName, string fusionLog, int hResult)
             _message = FormatFileLoadExceptionMessage(FileName, HResult);
         }
 
-        internal static string FormatFileLoadExceptionMessage(string fileName, int hResult)
+        internal static string FormatFileLoadExceptionMessage(string? fileName, int hResult)
         {
-            string format = null;
+            string? format = null;
             GetFileLoadExceptionMessage(hResult, JitHelpers.GetStringHandleOnStack(ref format));
 
-            string message = null;
+            string? message = null;
```

Comment:
Seems a shame to make it nullable when it will be definitely assigned through one path or the other. I suppose you could do
```c#
string message = SR.Arg_BadImageFormatException;
if (hResult != System.HResults.COR_E_BADEXEFORMAT)
    GetMessageForHR(hResult, JitHelpers.GetStringHandleOnStack(ref message));
```
Not sure whether that's better, but it feels like expressiveness is missing.

Reply:
It shouldn't need to be nullable once GetStringHandleOnStack is attributed to guarantee that `ref message` is non-null upon return.
https://github.com/dotnet/roslyn/issues/26761#issuecomment-479035368
cc: @jcouv

Reply:
Ok, so it seems like we will wait until we have these attributes available to be able to attribute `GetStringHandleOnStack` correctly. So I'll go ahead and add a `//TODO-NULLABLE` comment on these guys.